monitoring/mongodb/alerts.test.yaml

```diff
@@ -258,5 +258,88 @@ tests:
             exp_annotations:
               description: MongoDb has low disk space
               summary: MongoDb has low disk space
+    
+  - name: MongoDbPodRecovering
+    interval: 1m
+    input_series:
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-mongos-0", rs_state="3"}
+        values: 3x60
+
+    alert_rule_test:
+      - alertname: MongoDbPodRecovering
+        eval_time: 1h
+        exp_alerts:
+          - exp_labels:
+              namespace: zenko
+              severity: warning
+              pod: data-db-mongodb-sharded-mongos-0
+              rs_state: 3
+            exp_annotations:
+              description: "MongoDB pod `data-db-mongodb-sharded-mongos-0` has been stuck in the 'RECOVERING' state for the past hour. The instance may not be able to join the replica set if the platform ingests a large number of operations during this time. This alert is expected if the 'Resync a Data Services MongoDB Member' procedure has recently been executed."
+              summary: MongoDB node recovering for too long
```

Comment:
Now that we have the new severity, we can test it here

Reply:
The test now is for both the warning and critical alerts https://github.com/scality/Zenko/pull/2142/commits/3adf378e693d439cd90e3fca3291eef1f0c3cee2#diff-94c90083eba829343ee5745d0056d887a02d888847ea33af739a3ef5c3ae92c6R275-R287